src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -282,7 +282,7 @@ private void RecursiveActivity1Stop(string SourceName, string EventName, IEnumer
         /// </summary>
         /// <param name="SourceName"></param>
         [Event(10, Keywords = Keywords.Events)]
-        private void NewDiagnosticListener(string SourceName)
+        private void NewDiagnosticListener(string? SourceName)
```

Comment:
Ditto for SourceName here, and perhaps anywhere else the pattern occurs below.

Reply:
This one also can or cannot be null, marked null as calling method accepting null, will revert back